lib/components/molecules/Form/DDFileInput/DDFileInput.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Label from '../../../atoms/Label';
+import Input from '../../../atoms/FileInput';
+import styles from './DDFileInput.style';
+import useOpenFileDialog from '../../../../core/utils/useOpenFileDialog';
```

Comment:
Place it under lib/Hooks/UseOpenFileDialog/

Reply:
This has been implemented.